src/website/index.html

```diff
@@ -235,6 +244,7 @@
     defaultZoomValueIfSupported: 1.5
   }, false)
   html5QrcodeScanner.render(onScanSuccess);  
+  html5QrcodeScanner.render(markAttendance);
```

Comment:
Instead of calling `html5QrcodeScanner.render` twice (which will cause side effects as this modifies HTML and creates DOM event listeners), I think the cleaner solution is to only call `onScanSuccess` as is and then from there call `markAttendance` after [this line](https://github.com/thedevelopnik/kuberoke-registration/pull/1/files/5fe7c0ea9349fe6060c9dbe285e11c0441c00923..66e0afa2341bf8b7036e295dca6a3b649da72dc9#diff-c14853fa6f8a9ccf320b403bb30866c7e7e2f0a3b647f4e7e7b5636a47e64032R193).

Reply:
I'm good with that.